ui/packages/consul-ui/app/filter/predicates/auth-method.js

```diff
@@ -0,0 +1,11 @@
+export default {
+  kind: {
+    kubernetes: (item, value) => item.Type === value,
+    jwt: (item, value) => item.Type === value,
+    oidc: (item, value) => item.Type === value,
+  },
+  source: {
+    local: (item, value) => item.TokenLocality === value || item.TokenLocality === '',
```

Comment:
Did we figure out if Consul ever responds with an empty value for TokenLocality?

Reply:
yes, it responds with empty `''`

Reply:
Is that the mocks where you see that empty value or consul itself?

Reply:
oh, right. I need to verify this with BE dev.